README.md

```diff
@@ -16,6 +16,8 @@ Role Variables
 
 `redis_port` port that the redis instance listens on.
 
+`node_hipache_version` version of node-hipache that is to be installed. Defaults to `latest`
```

Comment:
@dcarley do you mean by naming this variable as `hipache_node_hipache_variable`?

I checked the [anisble documentation about naming](https://docs.ansible.com/playbooks_variables.html#variables-defined-from-included-files-and-roles) and [ansible best practice documentation](http://docs.ansible.com/playbooks_best_practices.html#group-and-host-variables) abd they are pretty open on how you name them.